src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    autocomplete="off" onsubmit="onSubmit()">
```

Comment:
See checklist item 3 about attributes
https://github.com/mate-academy/layout_html-form/blob/master/checklist.md
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      autocomplete="off"
      onsubmit="onSubmit()"
    >
```